builtin/pwd.c

```diff
@@ -0,0 +1,14 @@
+#define _POSIX_C_SOURCE 200809L
+#include <stdio.h>
+#include <stdlib.h>
+#include <mrsh/shell.h>
+#include "builtin.h"
+
+// usage: pwd [-L|-P]
+
+int builtin_pwd(struct mrsh_state *state, int argc, char *argv[]) {
+  // TODO `-P` and `-L`
+  const char *pwd = mrsh_env_get(state, "PWD", NULL);
+  puts(pwd);
```

Comment:
This should have a NULL check, in case the user `unset`s it.

Reply:
Done